doc/man3/RAND_DRBG_reseed.pod

```diff
@@ -37,6 +38,9 @@ and mixing in the specified additional data provided in the buffer B<adin>
 of length B<adinlen>.
 The additional data can be omitted by setting B<adin> to NULL and B<adinlen>
 to 0.
+The B<prediction_resistance> flag requests a reseeding from a live entropy
+source, if it is not 0.
+Currently, this reseed will always fail.
```

Comment:
I would stay more closely to [your formulation in #8600](https://github.com/openssl/openssl/pull/8660/files#diff-ee7d80bd16cd1ba1b201daf3474aa022) and also provide a reason for why it fails, e.g.:

```
An immediate reseeding from a live entropy source can be requested by setting
the B<prediction_resistance> flag to 1.
This feature is currently not implemented yet, so reseeding with prediction resistance
requested will always fail.
```

Reply:
Changed.